.prettierignore

```diff
@@ -1,18 +1,16 @@
-node_modules/**/*
```

Comment:
This was listed twice, you can see that the second entry is preserved in this diff